relay-server/src/utils/native.rs

```diff
@@ -108,6 +111,21 @@ fn write_crashpad_annotations(
         contexts.insert("crashpad".to_owned(), Context::Other(crashpad_context));
     }
 
+    match minidump.get_stream::<StabilityReport>() {
+        Ok(stability_report) => {
+            contexts.add(StabilityReportContext::from(stability_report));
+        }
```

Comment:
Maybe only should do it if the context is missing, but I guess we also don't do it for the crashpad context.